File: QTD2.Infrastructure/Model/Locations/LocationLatestActivityVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.Locations
{
    public class LocationLatestActivityVM
    {
        public int LocId { get; set; }
        public string LocCategoryID { get; set; }
        public string LocNumber { get; set; }
        public string LocName { get; set; }
        public string ActivityDesc { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreatedDate { get; set; }

        public string ModifiedBy { get; set; }

        public DateTime? ModifiedDate { get; set; }
    

    }
}

File: QTD.SPA/src/app/_DtoModels/Locations/LocationLatestActivityVM.ts
export interface LocationLatestActivityVM{
    locId: number;

    locCategoryId: number;

    locNumber: string;

    locName: string;

    activityDesc: string;

    createdBy: string;
  
    createdDate: Date | string | null;
  
    modifiedBy: string;
  
    modifiedDate: Date | string | null;
}